inc/functions.php

```diff
@@ -87,3 +87,51 @@ function core_sitemaps_get_max_urls( $type = '' ) {
 	 */
 	return apply_filters( 'core_sitemaps_max_urls', CORE_SITEMAPS_MAX_URLS, $type );
 }
+
+if ( ! function_exists( 'esc_xml' ) ) :
+	/**
+	 * Escaping for XML blocks.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @param string $text
```

Comment:
Parameter needs a description, e.g. "Text to be escaped"

Reply:
```suggestion
	 * @param string $text Text to escape
```

Reply:
this and the other PHPCS changes you suggested are now done.